inst/package_report/package_vignette_template.Rmd

```diff
@@ -0,0 +1,61 @@
+---
+title: "{{pkg_name}} Package Report, by pkgnet"
+output: 
+    rmarkdown::html_vignette:
+        toc: true
+vignette: >
+    %\VignetteIndexEntry{Package Report, by pkgnet}
+    %\VignetteEngine{knitr::rmarkdown}
+    %\VignetteEncoding{UTF-8}
+---
+
+This report on **{{pkg_name}}** is generated by [**pkgnet**](https://uptakeopensource.github.io/pkgnet/), an R package for analyzing other R packages through the lens of [network theory](https://en.wikipedia.org/wiki/Network_theory). 
+
+```{r setup, include=FALSE}
```

Comment:
Do you need an `echo=FALSE` on this block? Since you aren't setting that option until further down?

I don't know what `include=FALSE` does

Reply:
`include=FALSE` evaluates the code and hides everything. I think we use this in the other Rmd too.

Yihui has a whole blog post about 

> I added an option named include in knitr almost from Day 1 (thanks to Sweave), but I feel it is often ignored by users

https://yihui.name/en/2017/11/knitr-include-false/

😆

Reply:
amazing. I'm leaning a lot reviewing this!